manifest.json

```diff
@@ -2,16 +2,16 @@
     "spark_version": "2.4.0",
     "default_spark_dist": {
         "hadoop_version": "2.9",
-        "uri": "https://downloads.mesosphere.com/spark/assets/2.8.0/spark-2.4.0-bin-hadoop2.9.2.tgz"
+        "uri": "https://downloads.mesosphere.com/spark/assets/spark-2.4.0-bin-2.9.2-openjdk8-SNAPSHOT-20190530.tgz"
```

Comment:
Please, re-upload artefacts to follow the pattern. `hadoop` suffix should be present to make the numbers clear. Tarball name should have a form `spark-2.4.0-bin-hadoop<hadoop version>-openjdk8-SNAPSHOT-<YYYYmmDD>`

Reply:
ok, I fixed this, but such names were generated during the packages assembling. So, to avoid such naming issues in future, perhaps, we can adjust packages name generation algorithm, which uses during Spark packages building, to comply with this pattern?

Reply:
This is actually a good question. We're adding `hadoop` suffix to make it more readable for the customers using Spark. However, it is different from the upstream distribution which is more widely used. Changing the final name in the code doesn't seem right, because it will require upstreaming it which will change the standard naming for all the Spark users, rolling it back will mean that we'll be cherry-picking that commit forever. I think it's better to address it with build tooling.

Reply:
Yes, once we will have appropriate CI Job for build Spark package, we can rename the archive with convenient format.